Packet++/src/WireGuardLayer.cpp

```diff
@@ -0,0 +1,149 @@
+#define LOG_MODULE PacketLogModuleWireGuardLayer
+
+#include "UdpLayer.h"
+#include "IPv4Layer.h"
+#include "IPv6Layer.h"
+#include "WireGuardLayer.h"
+#include "Logger.h"
+#include <sstream>
+#include <iomanip>
+#include <cstring>
+
+namespace pcpp
+{
+	std::string WireGuardLayer::toString() const
```

Comment:
We try to make `toString()` output only one line, so we can print a full packet with one row per layer